openfisca_france/model/prestations/prestations_familiales/base_ressource.py

```diff
@@ -168,7 +166,9 @@ def nb_enf(famille, period, age_min, age_max):
     Renvoie le nombre d'enfant au sens des allocations familiales dont l'âge est compris entre ag1 et ag2
     """
 
-    period = period.this_month
+    assert period.unit == u'month'
+    assert period.size == 1
```

Comment:
Oops, I did not see that because of the diff display. All right then.

I don't want to open here a debate on helpers vs variables, but we're right in here.